da/go.mod

```diff
@@ -1,32 +1,39 @@
 module github.com/evstack/ev-node/da
 
-go 1.24.1
+go 1.24.6
 
 require (
+	github.com/celestiaorg/go-square/v3 v3.0.2
+	github.com/evstack/ev-node v1.0.0-beta.10
```

Comment:
can we avoid this import? it will create a circular dep in places and tagging a nightmare